modules/nf-core/gatk4/denoisereadcounts/main.nf

```diff
@@ -0,0 +1,57 @@
+process GATK4_DENOISEREADCOUNTS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::gatk4=4.4.0.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gatk4:4.4.0.0--py36hdfd78af_0':
+        'biocontainers/gatk4:4.4.0.0--py36hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(counts)
+    tuple val(meta2), path(pon)
+
+    output:
+    tuple val(meta), path("*_standardizedCR.tsv"), emit: standardized
+    tuple val(meta), path("*_denoisedCR.tsv")    , emit: denoised
+    path "versions.yml"                          , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    def avail_mem = 3072
+    if (!task.memory) {
+        log.info '[GATK CondenseDepthEvidence] Available memory not known - defaulting to 3GB. Specify process memory requirements to change this.'
+    } else {
+        avail_mem = (task.memory.mega*0.8).intValue()
+    }
+    """
+    gatk --java-options "-Xmx${avail_mem}M" DenoiseReadCounts \\
+        --input ${counts} \\
```

Comment:
```suggestion
        ${args} \\
        --tmp-dir . \\
        --input ${counts} \\
```

args again and added the temp dir to the current directory (this to make sure a limited temp folder does not cause the pipeline to fail)